src/main/java/com/ericsson/ei/EndpointSecurity.java

```
@@ -123,4 +118,19 @@ private void addLDAPServersFromList(JSONArray serverList, AuthenticationManagerB
     private boolean checkIfPasswordEncrypted(final String password) {
         return (password.startsWith("ENC(") && password.endsWith(")"));
     }
+    
+    private String decryptPassword(String password) {
+        TextFormatter textFormatter = new TextFormatter();
+        StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
+
+        if (jasyptEncryptorPassword.isEmpty()) {
+            LOGGER.error("Property -jasypt.encryptor.password need to be set for decrypting LDAP password.");
```

Should we throw an Exception instead here?